applications/nrf_desktop/src/modules/usb_state.c

```diff
@@ -959,13 +1067,11 @@ static uint32_t get_idle_next(const struct device *dev, const uint8_t id)
 static void report_sent_cb_next(const struct device *dev)
 {
 	struct usb_hid_device *usb_hid = dev_to_usb_hid(dev);
-
+	struct usb_hid_buf *buf = usb_hid_buf_find(usb_hid, USB_HID_BUF_SENDING);
```

Comment:
Shouldn't we check that the buffers also have allocated status?

Reply:
Buffer marked as `USB_HID_BUF_SENDING` must always be marked as `USB_HID_BUF_ALLOCATED`. Still, asserting this should not hurt. Added an assertion to `report_sent`

Reply:
yes, but the assert should help a little bit to future-proof this change in case of future contributions